lib/commanded/aggregates/aggregate.ex

```diff
@@ -604,20 +625,10 @@ defmodule Commanded.Aggregates.Aggregate do
       snapshotting: snapshotting
     } = state
 
-    Logger.debug(describe(state) <> " recording snapshot")
-
-    state =
-      case Snapshotting.take_snapshot(snapshotting, aggregate_version, aggregate_state) do
-        {:ok, snapshotting} ->
-          %Aggregate{state | snapshotting: snapshotting}
-
-        {:error, error} ->
-          Logger.warning(describe(state) <> " snapshot failed due to: " <> inspect(error))
-
-          state
-      end
-
-    noreply_with_lifespan(state)
+    with {:ok, snapshotting} <-
```

Comment:
I like that you left `noreply_with_lifespan/1` out of here.

I think the single-case `with` should go back to being a `case` here.

Reply:
I added a `case` back in there together with the logs in cfef270fb330ab3ba452d6a4d710aa88f4a46af7 🙇